src/xtgeo/grid3d/_grid_export.py

```diff
@@ -25,106 +23,7 @@
 
 def export_roff(self, gfile, option):
```

Comment:
Remove option

Reply:
I replaced it with a `roff_format` parameter so that the function now correctly exports to ascii when fformat specifies it.